about/strategy.md

```diff
@@ -0,0 +1,102 @@
+- Research teams within universities
+- University-wide education
+- Event-based commmunities of practice
+
+## Use-cases we'll focus on
+
+2i2c must understand the major use-cases that it can focus its infrastructure efforts around, in order to build a repeatable and scalable service.
+In the pilot, we will focus on a subset of use-cases that we believe are impactful across many communities of practice and organizations.
+
+- **Collaborative learning environments** - Communities of Practice that are focused around teaching and learning, and benefit from shared infrastructure to facilitate communicating and sharing with one another. Similar to our experience with Data 8, Syzygy, and Callysto.
+- **Scalable research environments** - Communities of Practice that use cloud infrastructure to scale their workflows - either by accessing large datasets or leveraging scalable computing infrastructure from an interactive session. Similar to our experience with the Pangeo project.
+- **Community event hubs** - Communities of Practice that have a time-bound event (e.g., a workshop or hackathon) that would benefit from a shared space to do their work and collaborate with one another. Similar to our experience with the NeuroHackademy and Pangeo workshops.
+
+## Infrastructure strategy for the pilot
+
+We make a few base assumptions about the kind of infrastructure we will focus on in this pilot.
+Here are several major components.
+
+### Where to deploy the infrastructure
+
+We'll focus our deployments on **major commercial cloud providers**.
+These are the most likely places for organizations to want to run cloud infrastructure, and are the most scalable and sustainable.
+In addition, getting Jupyter infrastructure to run well on all of the major cloud providers will have a large impact on a community's right to replicate.
+
+We'll focus on the following cloud providers:
+
+- Google Cloud
+- Amazon Web Services
+- Microsoft Azure
+
+### Why Jupyter and JupyterHub?
+
+- The Jupyter ecosystem is a collection of building blocks that are highly customizable and composable. They are popular and useful for many use-cases, but still require expertise to customize for a particular need. This is well-suited for 2i2c's skillset and the kind of service it wishes to provide.
+- Jupyter is a community-led and multi-stakeholder ecosystem that aligns well with 2i2c's commitment to vendor-agnosticity and the Right to Replicate
+- JupyterHub allows you to access centralized infrastructure for a community, but in a way that gives that community a lot of control over the details. It is a good balance between "SaaS" and "Fully bespoke community infrastructure". JupyterHub can be deployed via a single repository, but is also deployable by individual people or communities, providing them a clear off-ramp.
+
+
+## Major questions to answer
+
+Below are the major questions we'd like to answer with this pilot.
+
+- How much work does it take to manage a single JupyterHub?
```

Comment:
I think that is not the right question to ask... we are trying to find out something else, right?
We are trying to seize how much work do we need to do to maintain dozens of JupyterHubs... so maybe the question is: _How much work does it take to manage a community of JupyterHubs?_

Reply:
Well maybe both questions :wink:

Reply:
Good point - I'll add some more nuance to this one